spec/Integration/users/user_show_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe 'User pages functionality : user show page' do
+  let!(:user) do
+    User.create(name: 'Maqueen', photo: 'https://media.istockphoto.com/vectors/default-profile-picture-avatar-photo-placeholder-vector-illustration-vector-id1223671392?k=20&m=1223671392&s=612x612&w=0&h=lGpj2vWAI3WUT1JeJWm1PRoHT3V15_1pdcTn2szdwQ0=',
+                bio: 'Programer in headquaters', postscounter: 0)
+  end
+
+  let!(:p1) do
+    Post.create(user_id: user.id, title: 'Post test', text: 'Yes its just a test',
+                comments_counter: 0, likes_counter: 0)
+  end
+
+  before(:each) do
+    visit user_path(user)
+  end
+  it "I can see the user's profile picture" do
+    expect(page).to have_xpath("//img[@src='#{user.photo}']")
+  end
+  it "I can see the user's username" do
+    expect(page).to have_text(user.name)
+  end
+
+  it 'I can see the number of posts the user has written.' do
+    expect(page).to have_content(/posts:[\d+]/i)
+  end
+
```

Comment:
- One Test is missing, Please add it:
- **Has featured tests for `User` show page:** "Tests that asserts you can see the user's first 3 posts."

Reply:
@OLIPLICHE Thank you for the feedback !!